### DISCUSSION TITLE
Okay, so no adding collections in production mode. Any alternatives to manage content in a 'fashionable' way?

### DISCUSSION THREAD

**Discussion by Levisnkyyyy** [strapi/strapi#7580] [Legacy Topics]
Hey all!
Let's say I want to use Strapi as CMS for multiple restaurants (and for each restaurant there is a 'Categories' collection, and a 'Products').

Let's take one restaurant as an example and name it "Burgerz", so my feeling is to do it like this:
"Burgerz_Categories" and "Burgerz_Products" collection.
Now let's say I need this to happen automatically, with no manual intervention from my side. 
Restaurant owner signs up on a website -> 2 collections are created (the ones mentioned above) -> and so on.
But unfortunately it's not possible in Strapi Production to create collections.

Any other ways around? (or ideas!)
I just feel like putting ALL of the restaurants (Not just Burgerz!) products in one collection (with relations obviously) is really messy and not how it should be looking.

**Comment by derrickmehaffy** [comment]
Hi @Levisnkyyyy what it sounds like you are trying to do is [Multi-tenacy](https://portal.productboard.com/strapi/1-roadmap/c/36-multi-tenancy) which isn't something we support yet. In your case you would have a content-type for restaurants and one for products, along with any tags (could be used for both), but you are correct that content-types cannot be added in production and there are no real workarounds for that.

Content-types save their settings in files, so there really won't be a good way to automate this without any interaction from you.

**Reply by Levisnkyyyy** [reply]
Thank you!
